src/nostr-zap/zap-utils.ts

```
@@ -0,0 +1,207 @@
+  profile: string;
+  nip19Target?: string;
+  amount: number;
+  relays: string[];
+  comment?: string;
+  anon?: boolean;
+}) => {
+  const event = nip57.makeZapRequest({
+    profile,
+    event:
+      nip19Target && nip19Target.startsWith('note')
+        ? decodeNip19Entity(nip19Target)
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)